master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml

```diff
@@ -69,6 +69,11 @@ spec:
     spec:
       # Only run this pod on the master.
       tolerations:
+      # this taint is set by all kubelets running `--cloud-provider=external`
+      # so we should tolerate it to schedule the calico pods
+      - key: "node.cloudprovider.kubernetes.io/uninitialized"
```

Comment:
Just a nit, but the other tolerations don't have quotes around them, so for consistency let's drop the quotes around the strings.